specification/hugr.md

```diff
@@ -1168,44 +1168,78 @@ The new hugr is then derived as follows:
 
 ###### `Replace`
 
-This takes as input:
+This is the general subgraph-replacement method.
 
-  - a set S of IDs of nodes in Γ, all of which are separated;
+A _partial hugr_ is is a graph $G$ having the same node and edge types as a hugr
+and satisfying all the same local constraints except that it may be missing edges.
```

Comment:
Are "local constraints" defined somewhere? I mean, I get the general idea, but wondering if there is room for ambiguity. Or is what follows the precise definition of which constraints these do include?
(What follows certainly changed my initial reading - "empty container nodes" - so does "local constraints" mean those referring to value/order/static/controlflow edges?)

Reply:
Rather than "it may be missing edges" consider "some value (/controlflow??) ports may not have edges" (to/from/linked to/from/etc.)